packages/api/e2e/amino.spec.ts

```diff
@@ -0,0 +1,101 @@
+import { DeliverTxResponse } from '@cosmjs/stargate';
+import { RegenApi } from '../src/api';
+import {
+  MsgCreateClass,
+  MsgSend,
+} from '../src/generated/regen/ecocredit/v1/tx';
+import { StdFee } from '@cosmjs/amino/build/signdoc';
+import { Secp256k1HdWallet } from '@cosmjs/amino/build/secp256k1hdwallet';
+
+const TEST_ADDRESS = 'regen1m0qh5y4ejkz3l5n6jlrntxcqx9r0x9xjv4vpcp';
+const REDWOOD_NODE_TM_URL = 'http://redwood.regen.network:26657/';
+const TEST_FEE: StdFee = {
+  amount: [
+    {
+      denom: 'uregen',
+      amount: '5000',
+    },
+  ],
+  gas: '200000',
+};
+const TEST_MEMO = `regen-js v${process.env.npm_package_version} tests`;
+
+const connect = async (): Promise<RegenApi> => {
+  const mnemonic =
+    'time dice choose cabbage suit panic silly cattle picture auto grab hole';
+
+  const signer = await Secp256k1HdWallet.fromMnemonic(mnemonic, {
+    prefix: 'regen',
+  });
+
+  return RegenApi.connect({
+    connection: {
+      type: 'tendermint',
+      endpoint: REDWOOD_NODE_TM_URL,
+      signer,
+    },
+  });
+};
+
+describe('RegenApi with tendermint connection', () => {
```

Comment:
These two tests pass when run with @clevinson 's [cosmjs fork](https://github.com/regen-network/cosmjs/pull/1), which removes the top-level MsgType key. This PR won't pass the tests until that fix is published and referenced in this PR package.json.